app/routes/events/components/EventDetail/index.js

```diff
@@ -328,6 +328,18 @@ export default class EventDetail extends Component<Props> {
                     />
                   )}
 
+                  {event.useContactTracing && !currentRegistration && (
+                    <div>
+                      <i className="fa fa-exclamation-circle" /> Ved å melde deg
+                      på dette arrangementet samtykker jeg til at
+                      kontaktinformasjonen min (navn, telefonnummer og epost)
+                      kan deles med FHI og NTNU for smittesporing.
+                      Kontaktinformasjonen vil være tilgjengelig for brukeren
+                      som laget arrangementet i 14 dager etter at arrangementet
+                      har funnet sted, og vil kun brukes til smittesporing.
+                    </div>
```

Comment:
This sentence has some subject issues. I suggest changing _jeg_ to _du_.

We say in the `exportInfo` that if the information is to be shared with people outside Abakus, we should inform them of who this is. Should we not say something like "og eventuelle andre aktører nevnet i beskrivelsen av arrangementet"? So that we can share the info to the owners of the various locations.